drivers/google/compute_util.go

```diff
@@ -277,6 +275,17 @@ func (c *ComputeUtil) createInstance(d *Driver) error {
 	return nil
 }
 
+// tags computes the tags for the instance.
+func tags(d *Driver) []string {
```

Comment:
Why not make this a method directly on the driver and name it something slightly more descriptive (i.e. `func (d *Driver) parseTags() []string` ?